src/windows/staff.c

```diff
@@ -1325,6 +1325,17 @@ void window_staff_viewport_init(rct_window* w){
 	window_invalidate(w);
 }
 
+void staff_get_valid_skins(int* ebx)
+{
+	init_scenery();
+	*ebx = 0;
```

Comment:
It seems to use ```window_scenery_tab_entries``` which is only initialised in ```init_scenery()```. Without ```init_scenery()```, calling it might yield incorrect results. But ```init_scenery()``` also seems overkill, its almost like it should just take a part of ```init_scenery()```.